modules/services/osmscout-server.nix

```
@@ -0,0 +1,89 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.services.osmscout-server;
+in {
+  meta.maintainers = [ maintainers.Thra11 ];
+
+  options = {
+    services.osmscout-server = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Whether to enable OSM Scout Server as a systemd service.
+        '';
+      };
```

Should be sufficient with

```suggestion
      enable = mkEnableOption "OSM Scout Server";
```